commands/cmdsets/home.py

```diff
@@ -1121,7 +1121,7 @@ def pay_owner(self, price, msg):
         loc = self.caller.location
         loc.db.shopowner.pay_money(-price)
         assets = loc.db.shopowner.player_ob.assets
-        if price >= assets.min_silver_for_inform:
+        if price - (price * (self.get_discount() / 100.0)) >= assets.min_silver_for_inform:
```

Comment:
I believe the price here is already discounted. Did you mean `+` rather than `-`, so it'd use the undiscounted price for sending an inform? I'm not sure I follow what the intent is here. 

A best practice is often to make different commits and even separate pull requests for unrelated changes: it makes it easier to see what your intent is during code reviews and helps in version control to identify when things changed and how.

Reply:
I didn't realize this pushed through. I was testing an edit based on a bug request, didn't think it would append to my existing pull request.

Reply:
Ah! Okay, so the deal is that every pull request is tied to a branch: when you make a pull request, you're comparing the set of changes of commits on one branch to a branch you want to merge them into. Once the PR is opened, any further commits on that branch are added to it. To make a separate pull request, you'd make a different branch with the new set of changes.

Reply:
So while I have a pending pull request in, all the updates to my own git automatically alter my pull request in process. I didn't know that before, but good info. I'll be mindful of this moving forward.

Reply:
Well, on that branch: you can have multiple branches coexisting simultaneously and switch between them. It'd be like:
```
git checkout stable_orphan
git pull upstream
git checkout -b fix-something
git commit -am "Fixed thing!!"
git push -u origin fix-something
git checkout master
git checkout -b fix-other-thing
git commit -am "Fixed other thing!"
git push -u origin fix-other-thing
```
That'd get you two independent branches with different histories/changes.

Reply:
I'm not seeing that flow of commands, since I'm using Github Desktop. I have forsaken my command prompt origins.